phpcs.xml.dist

```diff
@@ -72,6 +72,11 @@
 	<rule ref="PHPCompatibility.FunctionDeclarations.NewReturnTypeDeclarations.boolFound">
 		<exclude-pattern>tests/utils/*</exclude-pattern>
 	</rule>
+
+	<rule ref="SlevomatCodingStandard.Functions.StaticClosure">
+		<exclude-pattern>tests/*</exclude-pattern>
```

Comment:
Is there a reason to exclude tests? Not a big deal either way, but this seems like a rule easy enough to follow that we could also apply it to tests. Or did you exclude because it would result in a ton more to fix in this first PR?

Reply:
Yeah, the latter. There are a lot more instances in tests. Happy to apply the changes there too if you like.

Reply:
Either is fine, maybe we can do that in a follow up PR instead.

Reply:
Went ahead and did it: f8b62ef